mlflow/keras/__init__.py

```diff
@@ -211,15 +253,19 @@ def predict(self, dataframe):
         return predicted
 
 
-def _load_pyfunc(path):
+def _load_pyfunc(path, keras_module=None):
     """
     Load PyFunc implementation. Called by ``pyfunc.load_pyfunc``.
 
     :param path: Local filesystem path to the MLflow Model with the ``keras`` flavor.
     """
-    import keras.backend as K
-    import tensorflow as tf
-    if K._BACKEND == "tensorflow":
+    if keras_module is None:
+        import keras
+        keras_module = keras
+
+    K = importlib.import_module(keras_module.__name__ + ".backend")
+    if keras_module.__name__ == "tensorflow.keras" or K._BACKEND == 'tensorflow':
```

Comment:
Does `tf.keras` refer to `_BACKEND` by some other name? (e.g., `K._BACKEND != 'tensorflow'`?)

Reply:
I think tf.keras always has backend tensorflow.